docs/docs/recipes/configure-connectors/email-connector/README.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_position: 1
+sidebar_label: Email connector
+---
+
+# Configure Email Connector
+
+Configuring an Email connector allows you to send a one-time verification code (OTP) to the user's email address ( Magic link will be supported in the future ). Email connector usage is similar to SMS connector, which can be utilized in various scenarios, including sign-up, sign-in, forgot password, and link-account processes, to validate the user's identity.
```

Comment:
OTP should stand for "one-time password". Gao made it clear in our official website yesterday.